app/src/main/java/com/example/keniphhomemaintenance/dwelling_screen/DwellingView.kt

```diff
@@ -0,0 +1,48 @@
+package com.example.keniphhomemaintenance.dwelling_screen
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import com.example.keniphhomemaintenance.navigation.Screen
+
```

Comment:
Dummy view so I could test and navigate to the Maintenance Item Screen. Actual implementation being handled by NAP

Reply:
@NickPerozzi don't resolve conversations that are context-providing :)